app/controllers/adventures_controller.rb

```diff
@@ -1,7 +1,6 @@
 class AdventuresController < ApplicationController
-  
-
   def create
+<<<<<<< HEAD
```

Comment:
Looks like there may be some leftover merge conflict delimiters in this file. Are these showing up locally too, or is this a GitHub thing?